Website/index.js

```diff
@@ -67,16 +139,35 @@
   return [year, month, day].join("-");
 }
 
-function blinkText() {
-  blink = true;
-  $(".icon-blink").each(function () {
-    var elem = $(this);
-    setInterval(function () {
-      if (elem.css("visibility") === "hidden") {
-        elem.css("visibility", "visible");
-      } else {
-        elem.css("visibility", "hidden");
+function checkUpdate() {
+  if (online) {
+    $.getJSON(`${baseURL}version`, function (data) {
+      if (data.version !== $("#version").text()) {
+        $("#updateBanner").fadeIn();
       }
-    }, 700);
   });
-}
+  }
+}
+
+if (online) {
+  $("#offlineBanner").hide();
+}
+
+function updateOnlineStatus() {
+    online = navigator.onLine ? true : false;
+    if (!online) {
+      $("#updateBanner").fadeOut();
+      $("#offlineBanner").fadeIn();
+    } else {
+      $("#offlineBanner").fadeOut();
+      checkUpdate();
+    }
+}
+
+checkUpdate();
+$("#updateBanner").hide();
+
+setInterval(function(){ checkUpdate(); }, 300000)
+
+  window.addEventListener('online',  updateOnlineStatus);
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Davianlols6/SC-Duty-List/pullRequest?prid=8406024)
```suggestion
  window.addEventListener("online",  updateOnlineStatus);
```